modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -8,9 +8,43 @@ source "${JBOSS_CONTAINER_MAVEN_S2I_MODULE}/maven-s2i"
 # include our s2i_core_*() overrides/extensions
 source "${JBOSS_CONTAINER_JAVA_S2I_MODULE}/s2i-core-hooks"
 
+# include our jlink scripts
+source "${JBOSS_CONTAINER_JAVA_JLINK_MODULE}/mkdeps"
```

Comment:
results in `I1011 11:58:23.526500  615171 sti.go:715] /usr/local/s2i/assemble: line 12: /opt/jboss/container/java/jlink/mkdeps: No such file or directory` (just missing `.sh` suffix)

Reply:
path is slightly wrong too (currently the script gets copied to `opt/jboss/container/java/s2i/mkdeps.sh`)